Taskfile.yml

```diff
@@ -50,6 +50,13 @@ tasks:
         FEATURE_TESTS: OFF
         CMAKE_BUILD_TYPE: Debug
 
+  cross_build_mingw:
+    - task: build_template
+      vars:
+        FEATURE_TESTS: OFF
+        CMAKE_BUILD_TYPE: Release
+        CONFIGURE_FLAGS: -DVCPKG_CHAINLOAD_TOOLCHAIN_FILE="/home/cmake/toolchains/x86_64-w64-mingw32.toolchain.cmake" -DVCPKG_TARGET_TRIPLET="x64-mingw-dynamic"
```

Comment:
Is this path always correct?

Reply:
No, this just for testing in the Docker container.

Reply:
So I guess we should handle it in project options

https://github.com/aminya/project_options/issues/140#issuecomment-1283494365